### DISCUSSION TITLE
OVAL CVE Audits

### DISCUSSION THREAD

**Discussion by jorx-dev** [uyuni-project/uyuni#8657] [Devel]
Hello, I've seen CVE OVAL audits in 2024.03 will improve detecting CVEs in systems that relied only on information that was found in their repository metadata. I guess this is true from now on for SCC SUSE products. Are there any plans to implement this feature for Ubuntu, Debian or AlmaLinux distributions which already provide those OVAL files in their web sites ?
Thank you !

**Comment by rjmateus** [comment]
The plan is to support also debian base and EL based systems.

**Comment by RenownTwiz** [comment]
The 2025.05 documentation doesn't list Oracle Linux as available for OVAL Sources.  Any reason why it wasn't included?  Seems like Oracle has posted their OVAL files here:  https://linux.oracle.com/security/oval/

**Reply by rjmateus** [reply]
The parser needs to be implemented, and we didn't had time to make it yet. We started with SUSE and RHEL. Then in a second step Ubuntu. 
Would you like to contribute to the implementation? It's java code and I can provide guidance :)

**Reply by RenownTwiz** [reply]
Not ignoring your response, just been really busy recently.  I'll have to look for the development build environment documentation.  I've not done java programing but have done some coding in years past in c/c++.  I'd want to get the build/dev environment stood up and have a look before committing to something that is over my head.
